### PR TITLE
LPS-166764 Add power function in ObjectCodeEditorUtil class

### PR DESCRIPTION
**Pull request by seltong**
Related issue: https://issues.liferay.com/browse/LPS-166764

### CODE REVIEW

**Comment by seltong** [comment]
ci:test:relevant

**Review by marcelabc — APPROVED** [review]
LGTM 🎉

**Comment by seltong** [comment]
ci:forward

**Comment by seltong** [comment]
ci:test:relevant

File: modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/object/definitions/display/context/util/ObjectCodeEditorUtil.java [hunk 1]
```diff
@@ -202,6 +207,13 @@ public static List<HashMap<String, String>> getItems(Locale locale) {
 			List<HashMap<String, String>> values = new ArrayList<>();
 
 			for (DDMExpressionFunction ddmExpressionFunction : values()) {
+				if (StringUtil.equals(ddmExpressionFunction._key, "power") &&
+					GetterUtil.getBoolean(
+						PropsUtil.get("feature.flag.LPS-164948"))) {
```
Comment (MarinhoFeliphe) [inline modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/object/definitions/display/context/util/ObjectCodeEditorUtil.java:212]: @seltong 

**!**`GetterUtil.getBoolean(PropsUtil.get("feature.flag.LPS-164948"))`

**Review by MarinhoFeliphe — CHANGES_REQUESTED** [review]

**Review by MarinhoFeliphe — APPROVED** [review]

**Comment by seltong** [comment]
ci:test:sf

**Pull request closed without merging**